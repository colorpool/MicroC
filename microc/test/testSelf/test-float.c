int main(){
    float i;
    i=233.23;
    print 'f' i;
}