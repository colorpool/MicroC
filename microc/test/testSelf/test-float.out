24 19 0 5 25 15 1 13 0 0 1 26 1130969825 12 15 -1 13 0 0 1 11 31 15 -1 15 -1 21 -1